src/chain/store/chain_store.rs

```diff
@@ -247,9 +248,9 @@ where
         file.insert(*cid);
     }
 
-    pub fn unmark_block_as_validated(&self, cid: &Cid) {
+    pub fn unmark_block_as_validated(&self, cid: Cid) {
```

Comment:
I'm not sure why this change is here.

Reply:
Honestly I don't remember either, possibly a leftover artifact from one of the previous iterations of this PR.

Reply:
Removed.